When all users of a domain are displayed, print a new line to recognize where each user's info starts/ends

diff -ruN vpopmail-5.4.33-original/vuserinfo.c vpopmail-5.4.33/vuserinfo.c
--- vpopmail-5.4.33-original/vuserinfo.c	2011-02-28 18:00:45.000000000 +0100
+++ vpopmail-5.4.33/vuserinfo.c	2022-09-12 17:55:25.672013594 +0200
@@ -110,6 +110,8 @@
 	}
 	while( (mypw=vauth_getall(Domain, first, 1))) {
 		first = 0;
+        /* When all users of a domain are displayed, print a new line to recognize where each user's info starts/ends */
+        printf("------------------------------------------\n");
 		/* display each user in the domain */
 		display_user(mypw, Domain);
 	}
